multiqc/modules/qualimap/QM_BamQC.py

```diff
@@ -505,7 +505,6 @@ def general_stats_headers (self):
         'title': 'Ins. size',
         'description': 'Median insert size',
         'min': 0,
-        'suffix': 'bp',
```

Comment:
Sorry - tiny thing: removing this is fine, but please add back to the description in that case so that the info is still there